src/qt/macdockiconhandler.mm

```diff
@@ -44,3 +43,12 @@ void setupDockClickHandler() {
 {
     delete s_instance;
 }
+
+/**
+ * Force application activation on macOS. With Qt 5.4 this is required when
```

Comment:
Given that we don't support Qt 5.4 anymore, can you update this comment? 
Obviously this workaround is still required with Qt 5.5+. Is there a Qt version where it's no-longer necessary?

Reply:
That would indeed be useful to know. I didn't try removing the workaround altogether.

Reply:
>  Is there a Qt version where it's no-longer necessary?

Looking through the Qt code base:
- since Qt 5.3:
https://github.com/qt/qtbase/blob/5.3/src/plugins/platforms/cocoa/qcocoaintegration.mm#L299

- since Qt 5.6:
https://github.com/qt/qtbase/blob/5.6/src/plugins/platforms/cocoa/qcocoaapplicationdelegate.mm#L299

- since Qt 5.8:
https://github.com/qt/qtbase/blob/5.8/src/plugins/platforms/cocoa/qcocoawindow.mm#L1013

I suppose the workaround is not needed on Qt 5.6+. But I have no capabilities to test this hypothesis now.

> I didn't try removing the workaround altogether.

I did. It works as expected on Catalina (Beta 3) + Qt 5.13.

Reply:
Our minimum QT version is 5.5.1, so maybe we should make a Github issue to consider dropping this workaround at the next bump https://github.com/bitcoin/bitcoin/blob/master/doc/dependencies.md